src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayload.cs

```diff
@@ -169,7 +188,7 @@ public void SetTransactions(params Transaction[] transactions)
 
     public virtual ValidationResult ValidateParams(IReleaseSpec spec, int version, out string? error)
     {
-        int GetVersion() => Withdrawals is null ? 1 : 2;
+        int GetVersion() => Withdrawals is null ? 1 : (BlobGasUsed is null && ExcessBlobGas is null && ParentBeaconBlockRoot is null ? 2 : 3);
```

Comment:
Cosmetic, consider improving readability of this :)

Reply:
Fixed